### PR TITLE
Set max line 100 characters

### PR DESCRIPTION
**Pull request by miry**
Add linter to check the max line length.

Use https://github.com/segmentio/golines to format most of files.

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -6,4 +6,5 @@ linters:
 
 linters-settings:
   lll:
-    line-length: 200
+    line-length: 100
+    tab-width: 1
```
Comment (lilien1010) [inline .golangci.yml:10]: how about adding `statements:75` too

**Review by lilien1010 — APPROVED** [review]
LGTM

File: .golangci.yml (hunk 1 above)
Comment (miry) [reply .golangci.yml:10]: I could not find this settings in https://github.com/golangci/golangci-lint/blob/2ea496f22b997490dd43bbb1536f28908b4389c6/pkg/config/linters_settings.go#L337

Comment (miry) [reply .golangci.yml:10]: I think it is from number of lines for FunLength: https://github.com/golangci/golangci-lint/blob/2ea496f22b997490dd43bbb1536f28908b4389c6/pkg/config/linters_settings.go#L200

**Pull request merged**